bindsnet/conversion/conversion.py

```diff
@@ -82,8 +82,8 @@ def forward(self, x: torch.Tensor) -> Dict[nn.Module, torch.Tensor]:
 class SubtractiveResetIFNodes(nodes.Nodes):
     # language=rst
     """
-    Layer of `integrate-and-fire (IF) neurons <http://neuronaldynamics.epfl.ch/online/Ch1.S3.html>`_ using reset by
-    subtraction.
+    Layer of `integrate-and-fire (IF) neurons
+    <http://neuronaldynamics.epfl.ch/online/Ch1.S3.html>` using reset by subtraction.
```

Comment:
the link is broken

Reply:
It works for me. Maybe the website was down temporarily?